### PR TITLE
[T4A5][W11-B3] Fan Weiguang

### PR DESCRIPTION
**Pull request by fanwgwg**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+		return this.person;
+	}
+	
+	public Tag getTag() {
+		return this.tag;
+	}
+	
+	public Behaviour getBehaviour() {
+		return this.behaviour;
+	}
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/tag/Tagging.java:33]: You forgot to write a method to format the string here, see #807 or #1198 .

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+/**
+ * Represents adding or deleting of a tag for a specific person that happened
+ * during current session.
+ *
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/tag/Tagging.java:8]: Good job for adding the header comment

**Comment by jeffryhartanto** [comment]
Some comments added. Please close the PR after reading comments. According to the diagram you should have initiated the Tagging object in the AddressBook class, see #1198 .

**Comment by fanwgwg** [comment]
Comments reviewed and I have already modified the code.

**Pull request closed without merging**